src/olympia/versions/models.py

```diff
@@ -734,6 +734,22 @@ def is_unreviewed(self):
     def sources_provided(self):
         return bool(self.source)
 
+    def flag_if_sources_were_provided(self, user):
```

Comment:
Do we need a test in test_models.py for this?  ("no we don't" is an acceptable answer 😄)

Reply:
https://github.com/mozilla/addons-server/pull/20543/commits/e2d9d44b9376cf04116115a79eacb0992ec9f6b2